experimental/devices/epd/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package epd controls Waveshare e-paper series displays
+//
+// More details
+//
+// See https://periph.io/device/epd/ for more details about the device.
+//
+// Based on https://github.com/soonuse/epd-library-python
+//
+// Product page:
```

Comment:
Please ensure this looks nice in godoc, you can see locally with `godoc -http=localhost:6060` then navigate to http://localhost:6060/pkg/periph.io/x/periph/experimental/devices/epd